tests/src/test/java/alluxio/master/journal/ufs/UfsJournalIntegrationTest.java

```diff
@@ -400,6 +403,9 @@ private void directoryTestUtil(URIStatus status) throws Exception {
     registry.stop();
   }
 
+  /**
+   * Tests directory persisting after creation.
```

Comment:
this test is testing that journalling works correctly when we (1) create a directory with MUST_CACHE, then (2) create the directory again with CACHE_THROUGH and AllowExists=true.